engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -576,6 +576,12 @@ public void startApplication() throws YarnException, IOException
         vargs.add("-Dlog4j.debug=true");
       }
 
+      String loggerAppender = dag.getValue(LogicalPlan.LOGGER_APPENDER);
+      if (loggerAppender != null) {
+        vargs.add(String.format("-D%s=\"%s\"", LogicalPlan.LOGGER_APPENDER.getName(), loggerAppender));
```

Comment:
Changed the implementation to:

  public String getLongName()
  {
    return "apex." + getSimpleName().replaceAll("_",".").toLowerCase();
  }